lms/envs/test.py

```diff
@@ -454,10 +454,10 @@
 INSTALLED_APPS += ('milestones', )
 
 # MILESTONES
-FEATURES['MILESTONES_APP'] = True
+FEATURES['MILESTONES_APP'] = False
 
 # ENTRANCE EXAMS
-FEATURES['ENTRANCE_EXAMS'] = True
+FEATURES['ENTRANCE_EXAMS'] = False
```

Comment:
These lines can just be removed since they are defaulted to `False` in `common.py`.